README.md

```diff
@@ -142,5 +166,17 @@ Using the same front-end application and Rails API from the previous question, h
 If the Post creation is successful, the new Post should be printed to the browser console. Otherwise, an error message should be printed to the console.
 
 ```js
-// Your answer goes here...
+$.ajax({
+  url: 'http://localhost:3000/posts',
+  type: 'post',
+  dataType: 'json',
+  data: {
```

Comment:
Because there's a strong params method, it's going to require that we namespace our data in a `post:` key. So something like this...

```js
data: {
  post: {
    title: "My first post"
  }
}